### ISSUE TITLE
Error: doc.hasOwnProperty is not a function

### ISSUE THREAD

**Issue by Siyfion** [aldeed/meteor-collection2-core#8]
I'm getting the following error, but it's a bit ambiguous to say the least, any ideas as to what might be causing this? I have only just update to the v2.x version, but I don't *think* it's related.

```
W20161206-15:16:26.523(0)? (STDERR) TypeError: doc.hasOwnProperty is not a function
W20161206-15:16:26.523(0)? (STDERR)     at [object Object].doValidate (packages/aldeed:collection2-core/collection2.js:361:13)
W20161206-15:16:26.524(0)? (STDERR)     at [object Object].Mongo.Collection.(anonymous function) [as insert] (packages/aldeed:collection2-core/collection2.js:187:25)
W20161206-15:16:26.524(0)? (STDERR)     at insertLabel (imports/api/labels/schema.js:156:28)
W20161206-15:16:26.524(0)? (STDERR)     at resolveOrError (/Users/siyfion/Git/LabelLogicLive/node_modules/graphql/execution/execute.js:473:12)
W20161206-15:16:26.525(0)? (STDERR)     at resolveField (/Users/siyfion/Git/LabelLogicLive/node_modules/graphql/execution/execute.js:459:16)
W20161206-15:16:26.525(0)? (STDERR)     at /Users/siyfion/Git/LabelLogicLive/node_modules/graphql/execution/execute.js:247:20
W20161206-15:16:26.525(0)? (STDERR)     at /Users/siyfion/.meteor/packages/promise/.0.8.8.1xgcipj++os+web.browser+web.cordova/npm/node_modules/meteor-promise/fiber_pool.js:32:39

```

**Comment by xavcz** [comment]
Got it too when trying an insert. Worked around it by creating a new object at the moment to insert 🙄 
```js
Collection.insert({...document})
```

**Comment by jschlieber** [comment]
@xavcz 

Strange, thanks for providing this work around.

**Comment by aldeed** [comment]
published fix in 2.0.1
